query_V1

```diff
@@ -95,9 +222,22 @@
 			
 			
 			</div>
+			
+			<div class="footer" style="background-color: lightgrey; grid-row: 4; grid-column: 2;">
+				
+				<a href="query_search.php#top" class="bottom-to-top"><img src="images/icon_7.png" style="height: 6vw; width: 6vw; margin: 2vw 0vw 0vw 69vw;" draggable="false"></a>
+			
+			</div>
+
+		
 		
 		</div>
+		
+		<script type="text/javascript">
+			$('div:odd').css('background-color', 'red');
```

Comment:
Here I have trialed some javascript that will make every second field a red background colour.